features/provider/active_docs.feature

```diff
@@ -0,0 +1,27 @@
+Feature: ActiveDocs pages
+  As a provider
+  I want to manage my ActiveDocs
+
+  Background:
+    Given a provider is logged in
+      And the provider has 1 active doc
+
+  Scenario: Index renders with the proper elements
+    When I go to the provider active docs page
+     And the table should contain the API
+
+  Scenario: Edit and the redirect after failure render with the proper elements
+    When I go to the edit active docs page
+    When I try to update the active docs with invalid data
+    Then I should see the errors in the page
```

Comment:
This is not specific to active docs so it might seem reusable in another feature